types/event.proto

```diff
@@ -35,4 +35,7 @@ message Event {
 
   // Hooks describes the results of multiple hooks; if event is associated to hook execution.
   repeated Hook hooks = 6 [(gogoproto.nullable) = true];
+  
+  // LastOK displays last time this check was ok; if event status is 0 this is set to timestamp
+  int64 last_ok = 7 [(gogoproto.customname) = "LastOK"];
```

Comment:
In this case, we're looking at the Check's status. We'll want to throw LastOK in [here](https://github.com/sensu/sensu-go/blob/master/types/check.proto#L201).